test/airport.spec.js

```diff
@@ -0,0 +1,205 @@
+const { assertEquals } = require("./test-framework");
+const Airport = require("../src/Airport");
+
+function testLandingPlane() {
```

Comment:
Could have separate all the tests into individual files instead of the same file.